### DISCUSSION TITLE
OTA updates

### DISCUSSION THREAD

**Discussion by ulipo** [dchristl/esp32_nat_router_extended#63] [Ideas]
Would it be possible to have OTA updates, maybe uploading the binary from web interface or downloading the right build directly from the server (Tasmota style)?

**Comment by dchristl** [comment]
Hello @ulipo,

I alredy thought about it and it is on my list for improvements already. It is not so easy and fast to implement and it requires some internal changes to the partition. So it will be coming, but I can't tell you when.

**Comment by dchristl** [comment]
This is implemented with the [latest update](https://github.com/dchristl/esp32_nat_router_extended/releases/tag/v6.0.0)
